src/Symfony/Component/Notifier/composer.json

```diff
@@ -26,16 +26,25 @@
     },
     "conflict": {
         "symfony/http-kernel": "<4.4",
-        "symfony/firebase-notifier": "<5.2",
-        "symfony/free-mobile-notifier": "<5.2",
-        "symfony/mattermost-notifier": "<5.2",
-        "symfony/nexmo-notifier": "<5.2",
-        "symfony/ovh-cloud-notifier": "<5.2",
-        "symfony/rocket-chat-notifier": "<5.2",
-        "symfony/sinch-notifier": "<5.2",
-        "symfony/slack-notifier": "<5.2",
-        "symfony/telegram-notifier": "<5.2",
-        "symfony/twilio-notifier": "<5.2"
+        "symfony/discord-notifier": "<5.3",
+        "symfony/esendex-notifier": "<5.3",
+        "symfony/firebase-notifier": "<5.3",
+        "symfony/free-mobile-notifier": "<5.3",
+        "symfony/google-chat-notifier": "<5.3",
+        "symfony/infobip-notifier": "<5.3",
+        "symfony/linked-in-notifier": "<5.3",
+        "symfony/mattermost-notifier": "<5.3",
+        "symfony/mobyt-notifier": "<5.3",
+        "symfony/nexmo-notifier": "<5.3",
+        "symfony/ovh-cloud-notifier": "<5.3",
+        "symfony/rocket-chat-notifier": "<5.3",
+        "symfony/sendinblue-notifier": "<5.3",
+        "symfony/sinch-notifier": "<5.3",
+        "symfony/slack-notifier": "<5.3",
+        "symfony/smsapi-notifier": "<5.3",
+        "symfony/telegram-notifier": "<5.3",
+        "symfony/twilio-notifier": "<5.3",
+        "symfony/zulip-notifier": "<5.3"
```

Comment:
should be this instead:
```
        "symfony/firebase-notifier": "5.2.0",
        "symfony/free-mobile-notifier": "5.2.0",
        "symfony/mattermost-notifier": "5.2.0",
        "symfony/nexmo-notifier": "5.2.0",
        "symfony/ovh-cloud-notifier": "5.2.0",
        "symfony/rocket-chat-notifier": "5.2.0",
        "symfony/sinch-notifier": "5.2.0",
        "symfony/slack-notifier": "5.2.0",
        "symfony/telegram-notifier": "5.2.0",
        "symfony/twilio-notifier": "5.2.0"
```

Reply:
(with the new 5.2 bridges added to the list)

Reply:
OR, we remove them all.
Yes, that will allow Composer to install a 5.2.0 bridge with notifier >= 5.3.
But in practice, I don't think this will happen.